packages/vscode-extension/package.nls.json

```diff
@@ -366,6 +368,7 @@
     "teamstoolkit.taskDefinitions.command.startLocalTunnel.description": "Start the local tunneling service.\n See https://aka.ms/teamsfx-tasks/local-tunnel for details and how to customize the arguments.",
     "teamstoolkit.taskDefinitions.command.provision.description": "Execute provision lifecycle.\n See https://aka.ms/teamsfx-tasks/provision for details and how to customize the arguments.",
     "teamstoolkit.taskDefinitions.command.deploy.description": "Execute deploy lifecycle.\n See https://aka.ms/teamsfx-tasks/deploy for details and how to customize the arguments.",
+    "teamstoolkit.taskDefinitions.command.launchWebClient.description": "Launch Teams web client. \n See https://aka.ms/teamsfx-tasks/launch-web-client for details and how to customize the arguments.",
```

Comment:
Better exclude link in strings since it's not needed for localization

Reply:
Sure. will use placeholder for the link and resolve in the caller.